conf/channels.yml

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) rAthena Dev Teams - Licensed under GNU GPL
+# For more information, see LICENCE in the main folder
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Channel Database
+###########################################################################
+#
+# Channel Settings
+#
+###########################################################################
```

Comment:
I dont really like to break the structure that we worked on for just this database.
I would recommend that you come up with a format in the following style instead:
```
Body:
  - Name: x
    Type: Public, Private, Map, Ally (enum Channel_Type)
    ...
```
If some configurations are only supported by one and not by the other issue an error.